src/Microsoft.ApplicationInsights/Extensibility/W3C/W3CActivityExtensions.cs

```diff
@@ -1,385 +0,0 @@
-﻿namespace Microsoft.ApplicationInsights.Extensibility.W3C
-{
-    using System;
-    using System.ComponentModel;
-    using System.Diagnostics;
-    using System.Diagnostics.CodeAnalysis;
-    using System.Globalization;
-    using System.Linq;
-    using System.Text.RegularExpressions;
-    using Microsoft.ApplicationInsights.Channel;
-    using Microsoft.ApplicationInsights.DataContracts;
-    using Microsoft.ApplicationInsights.Extensibility.Implementation;
-
-    /// <summary>
-    /// Extends Activity to support W3C distributed tracing standard.
-    /// </summary>
-    [EditorBrowsable(EditorBrowsableState.Never)]
-    public static class W3CActivityExtensions
```

Comment:
we cannot remove it - it's public. It could proxy calls to real Activity API when make sense and be noop in other cases